configs/restorers/liif/liif_edsr_norm_c64b16_g1_1000k_div2k.py

```diff
@@ -0,0 +1,167 @@
+exp_name = 'liif_edsr_norm_c64b16_g1_1000k_div2k'
+scale_min, scale_max = 1, 4
+
+# model settings
+model = dict(
+    type='LIIF',
+    generator=dict(
+        type='EDSR',
+        in_channels=3,
+        out_channels=3,
+        mid_channels=64,
+        num_blocks=16),
+    imnet=dict(
+        type='MLPRefiner',
+        in_dim=64,
+        out_dim=3,
+        hidden_list=[256, 256, 256, 256]),
+    local_ensemble=True,
+    feat_unfold=True,
+    cell_decode=True,
+    data_mean=[0.4488, 0.4371, 0.4040],
+    data_std=[1., 1., 1.],
+    eval_bsize=30000,
+    pixel_loss=dict(type='L1Loss', loss_weight=1.0, reduction='mean'))
+# model training and testing settings
+train_cfg = None
+test_cfg = dict(metrics=['PSNR', 'SSIM'], crop_border=scale_max)
+
+# dataset settings
+scale_min, scale_max = 1, 4
+# dataset settings
+train_dataset_type = 'SRFolderGTDataset'
+val_dataset_type = 'SRFolderGTDataset'
+test_dataset_type = 'SRFolderGTDataset'
+train_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        io_backend='disk',
+        key='gt',
+        flag='unchanged',
```

Comment:
What the reason to use `unchanged`?

Reply:
`color` is ok, too. Will be modified to `color`.

Reply:
They are different, so its better to be clear and exact